### PR TITLE
Repositioned Edit Profile button to improve UI/UX

### PR DESCRIPTION
**Pull request by r-gio28**
## Description

Moved editProfile button to further improve UI/UX. Functionality of modal was not
 compromised

https://www.loom.com/share/813e5b6c1bb74dcbaff13bcd9e827f03

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by justinpeisker — APPROVED** [review]
This is great Gio, I agree, this looks better!

**Review by justinpeisker — APPROVED** [review]

**Review by BryceSlade — APPROVED** [review]
This looks great. Much better positioning for the Edit Profile button.

**Review by tarasshulhan — APPROVED** [review]

**Review by OrdainedRat — COMMENTED** [review]
I feel like this really impactful for how small of a change it is. It looks good. Keep it up!

**Review by phillipgalbraith — COMMENTED** [review]
Upper righthand corner creates a good pattern for future problems like this!  Why is the word Edit used exclusively?  It looks like the icon for edit is also used exclusively in another location.  It use both the word Edit and the edit icon at each location.

File: src/components/pages/Profile/RenderProfileContainer.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ const RenderProfileContainer = props => {
         </Col>
         <Col span={17}>
           <Title level={1}>{userData.name}</Title>
-          <Card title="My Profile">
+          <Card title="My Profile" extra={<EditProfile />}>
```
Comment (paulstgermain) [inline src/components/pages/Profile/RenderProfileContainer.js:65]: Really good example of researching and then leaning into the component framework we've chosen for use here in Labs - good work!

**Pull request merged**